src/data/javascript-quiz.ts

```diff
@@ -2084,6 +2084,36 @@ const javascriptQuiz = [
     Explanation:
       "Variable names cannot start with a number or any special character.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types"
+  },
+  {
+    Question: "If const a = 2 and then a = 5 what will be the result?",
+    Answer: "A TypeError ",
+    Distractor1: "A TypeError ",
+    Distractor2: "5",
+    Distractor3: "2",
+    Explanation:
+      "In JavaScript, a variable declared with the `const` keyword cannot be reassigned.",
+    Link: "https://www.w3schools.com/js/js_const.asp"
```

Comment:
we don't not accept links from w3schools.
please read the docs on which links we do accept
https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md#guidelines-for-adding-quiz-questions

Reply:
done